js/narrativeView.js

```diff
@@ -102,61 +102,63 @@ define([
       }
       this.setupEventListeners();
       this._isInitial = false;
-    },
+    }
 
-    calculateWidths: function() {
-      var itemCount = this.model.getChildren().length;
+    calculateWidths() {
+      const itemCount = this.model.getChildren().length;
       this.model.set({
         _totalWidth: 100 * itemCount,
         _itemWidth: 100 / itemCount
       });
-    },
+    }
 
-    resizeControl: function() {
-      var previousMode = this.model.get('_mode');
+    resizeControl() {
+      const previousMode = this.model.get('_mode');
       this.renderMode();
       if (previousMode !== this.model.get('_mode')) this.replaceInstructions();
       this.evaluateNavigation();
-      var activeItem = this.model.getActiveItem();
+      const activeItem = this.model.getActiveItem();
       if (activeItem) this.setStage(activeItem);
-    },
+    }
 
-    reRender: function() {
+    reRender() {
       if (this.model.get('_wasHotgraphic') && this.isLargeMode()) {
         this.replaceWithHotgraphic();
-      } else {
-        this.resizeControl();
+        return;
       }
-    },
+      this.resizeControl();
+    }
 
-    closeNotify: function() {
+    closeNotify() {
       this.evaluateCompletion();
-    },
+    }
 
-    replaceInstructions: function() {
+    replaceInstructions() {
       if (this.isLargeMode()) {
         this.$('.narrative__instruction-inner').html(this.model.get('instruction'));
-      } else if (this.model.get('mobileInstruction') && !this.model.get('_wasHotgraphic')) {
+        return;
+      }
+
+      if (this.model.get('mobileInstruction') && !this.model.get('_wasHotgraphic')) {
         this.$('.narrative__instruction-inner').html(this.model.get('mobileInstruction'));
       }
-    },
+    }
 
-    replaceWithHotgraphic: function() {
+    replaceWithHotgraphic() {
       if (!Adapt.componentStore.hotgraphic) throw "Hotgraphic not included in build";
-      var HotgraphicView = Adapt.componentStore.hotgraphic.view;
-
-      var model = this.prepareHotgraphicModel();
-      var newHotgraphic = new HotgraphicView({ model: model });
+      const HotgraphicView = Adapt.componentStore.hotgraphic.view;
```

Comment:
Could you switch this to:
```suggestion
      const HotgraphicView = Adapt.getViewClass('hotgraphic');
```
`Adapt.componentStore` is also deprecated. It has become `Adapt.store` as a registry for all content view+model combinations. 

We should now not access the store directly as it will prohibit any future changes to the storage mechanism, like the headache whilst switching from view only to view + model storage. We've got `Adapt.getViewClass`, `Adapt.getModelClass`, `Adapt.getViewName` and `Adapt.getModelName` as ways to interact with the `Adapt.store` giving a bit of separation between the API and storage mechanism.